test/utf8/rlexe.xml

```diff
@@ -15,6 +15,13 @@
   </test>
   <test>
     <default>
+      <files>OS_2977448.js</files>
+      <compile-flags>-Intl-</compile-flags>
```

Comment:
Yeah, actually this was originally under Intl tests and I moved it since it has nothing to do with Intl features (I added the -Intl- flag to double-check that it wasn't dependent on Intl). I can remove the tag.